scripts/deployment/setup_ovs.py

```diff
@@ -521,6 +521,9 @@ def install_devstack(node_ip, fixed_range, fixed_range_size, floating_range, mas
 {sub_branches}
 EOF
 fi
+cat <<EOF >>local.conf
```

Comment:
no need to do a cat ... structure here .. add the oneliner to the top part ... after SERVICE_TOKEN=rooter